src/components/LibraryInfo/index.js

```diff
@@ -12,15 +12,17 @@ governing permissions and limitations under the License.
 
 import libraryVersion from "../../constants/libraryVersion";
 
-const createLibraryInfo = () => {
+const createLibraryInfo = tools => {
```

Comment:
Use [object destructuing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#unpacking_properties_from_objects_passed_as_a_function_parameter) for the parameters instead of a single parameter. [This matches the pattern of other component creators](https://github.com/adobe/alloy/blob/main/src/components/DataCollector/index.js#L15).

For example,

```js
// Instead of this…
const createCommand = tools => {
  return { foo: tools.foo + tools.bar };
}

// …do this.
const createCommand2 = ({ foo, bar }) => {
  return { foo: foo + bar };
};